.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,22 @@
+name: Feature Request
```

Comment:
Can we change this to "Enhancement"? Then it would match the tag and also apply to improvements for existing features.